webapp/src/Service/ImportExportService.php

```diff
@@ -522,13 +526,20 @@ public function getResultsData(
 
             $rank      = $teamScore->rank;
             $numPoints = $teamScore->numPoints;
-            if ($rank <= $contest->getGoldMedals()) {
+            $skip      = false;
```

Comment:
We don't really need this skip variable right? If it is false we just subtract 0

Reply:
> We don't really need this skip variable right? If it is false we just subtract 0

If the team is not in the medal category, it should be given a rank or honors. So I use this variable below.

Reply:
But you set it to true exactly when `skippedTeams` is not 0. So you can use that check?

Reply:
`skippedTeams` counts the total number of skipped teams.
`skip` marks whether the team in the current for loop needs to be skipped.

Reply:
Ahhh now I see it